Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
+                 div.padded {  
+      padding-top: 50px;  
+      padding-right: 400px;  
+      padding-bottom: 0.25in;  
+      padding-left: 400px;  
+    }
+            #rcorners2 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: DodgerBlue;
+            float: left;
+            margin-right: 20px
+            
+                    }
+            #rcorners3 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: plum;
+            float: right;
+            margin-left: 20px
+                    }   
+            #rcorners4 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: orange;
+            float: left;
+            margin-right: 20px
+                    }        
+            #rcorners1 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color:red;
+            float: right;
+            margin-left: 20px
+                    }          
+
+                    .main {
+             color: black;
+             font-size: 16px;
+             font-family: Helvetica;
+             
+             }
+
+            .name {
+             color: dodgerblue;
+             font-size: 30px;
+             font-family: Helvetica;
+             
+             }
+            .name2 {
+             color: plum;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name3 {
+             color:orange;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name4 {
+             color: red;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+
+             img.rounded-corners {
+border-radius: 15px;
+}
+
+             .image-container {
```

Comment:
This is used appropriately, but make sure you understand flex-box and the box model.

Read: https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Box_model
Play: https://flexboxfroggy.com/
Read: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox;